tests/validator_test.go

```diff
@@ -329,4 +337,611 @@ var _ = Describe("Template validator", func() {
 			Expect(err).ToNot(HaveOccurred(), "SSP status should be deployed.")
 		})
 	})
+
+	Context("Validation rules tests", func() {
+		var (
+			vm       *kubevirtv1.VirtualMachine
+			vmi      *kubevirtv1.VirtualMachineInstance
+			template templatev1.Template
+		)
+		const (
+			TemplateNameAnnotation      = "vm.kubevirt.io/template"
+			TemplateNamespaceAnnotation = "vm.kubevirt.io/template-namespace"
+		)
+
+		BeforeEach(func() {
+			vmi = NewRandomVMIWithBridgeInterface(strategy.GetNamespace())
+			// Create a dummy template to make sure there's no value from an earlier test
+			// And dummy name & namespace for a valid object in Cleanup
+			template = templatev1.Template{}
+			template.Name = "testDummy"
+			template.Namespace = "testDummy"
+		})
+		AfterEach(func() {
+			err := apiClient.Delete(ctx, &template)
+			if !errors.IsNotFound(err) {
+				Expect(err).ToNot(HaveOccurred(), "Failed to delete Template")
+			}
+			err = apiClient.Delete(ctx, vm)
+			if !errors.IsNotFound(err) {
+				Expect(err).ToNot(HaveOccurred(), "Failed to Delete VM")
+			}
+		})
+
+		It("should create VM without template", func() {
+			vm = NewVirtualMachine(vmi)
+			Expect(apiClient.Create(ctx, vm)).ToNot(HaveOccurred(), "Failed to create VM")
+		})
+		It("be created from template with minimal rules", func() {
+			template = TemplateWithMinimalRule()
```

Comment:
For this test, the original test case in the template validator repository is probably wrong. Can you change the code so that it tests creating a VM from template with no rules? Instead of changing the description.